src/components/project/MembersTab.js

```diff
@@ -176,9 +176,17 @@ class MembersTab extends Component {
       return
     }
     if (owner) {
+      const isRemoveDisabled = _.values(this.props.projectMembers.owners.byId).length <= 1
```

Comment:
Variable name doesn't really describe what it is as well as I'd imagine. Consider isLastOwner or something more descriptive?

Reply:
Agreed. Or just add a comment above this line saying `// If only one owner remains, disable owner removal`